cluster/ci/config/prow/config.yaml

```diff
@@ -84,15 +85,6 @@ tide:
     - needs-ok-to-test
     - do-not-merge/work-in-progress
     - do-not-merge/hold
-  - repos:
-    - openshift/origin
```

Comment:
Please revert this change, we keep origin separate from the rest of the queries to make merge gating on rebases easier.

Reply:
> Please revert this change, we keep origin separate from the rest of the queries to make merge gating on rebases easier.

So something like #707's pulling out openshift/online-registration?  Is there a rule of thumb to decide whether a given repo should get its own section or not?  I'm not clear on why openshift/origin would need a separate section while the rest of the repositories in the previous section do not.

Reply:
When @openshift/sig-master want to land a kubernetes rebase in openshift/origin#master, we need to block merges on that branch and at the same time we don't want to be blocking merges elsewhere, hence origin has its own query.